app/templates/google-services.json

```diff
@@ -197,35 +197,32 @@
     },
     {
       "client_info": {
-        "mobilesdk_app_id": "1:729925342863:android:3fcb586559e496d9",
+        "mobilesdk_app_id": "1:768065119425:android:4e058a2bb3994794",
         "android_client_info": {
-          "package_name": "org.commcare.managed"
+          "package_name": "org.commcare.dalvik.debug"
         }
       },
       "oauth_client": [
         {
-          "client_id": "729925342863-n2hj7r0d62d51uk62ts3b2k2j67k44b4.apps.googleusercontent.com",
+          "client_id": "768065119425-hg10t0osvjqrt6pne3toaobabn5gobqs.apps.googleusercontent.com",
           "client_type": 3
         }
       ],
       "api_key": [
         {
-          "current_key": "AIzaSyDFDJv-bUIEUHuT_idSKb5QWmN2Aj6UUWw"
+          "current_key": "@apiKey@"
```

Comment:
Will these changes require any updates to our local configuration?

Reply:
This isn't a change, we've always been using this @apiKey@ pattern. Did you mean something besides that? This is only showing as a diff bc it looks like when you added the `org.commcare.managed` package to this file, you were using the API key directly instead of using the param value (which was probably not good.. we must have missed this)